README.md

```diff
@@ -62,11 +62,11 @@ Sampling of traces is required in environments with high traffic load to reduce
 By default, sampling rates are provided by the Datadog agent, and additional controls are available if necessary using configuration of `initial_sample_rate` and `initial_samples_per_second`.
 
 The `initial_sample_rate` can be set to a value between 0.0 (0%) and 1.0 (100%), and this is limited by the setting for `initial_samples_per_second` (default: 100).
-After that amount of sampled traces has been exceeded, the sampling rates provided by the Datadog agent are used.
+After that amount of sampled traces has been exceeded, traces will not be sampled.
```

Comment:
It's common enough that by "sampled" we mean "kept," i.e. "included in the sample," but do consider using an alternative to "not [...] sampled."